corehq/apps/dhis2/models.py

```diff
@@ -172,5 +197,19 @@ def put(self, path, data, **kwargs):
         return JsonApiRequest.json_or_error(response)
 
 
-# Just for now. Use Documents when we need to persist
-DataValue = namedtuple('DataValue', ('dataElement', 'period', 'orgUnit', 'value'))
+class AsDictMixin(dict):
+    def _asdict(self):
+        # Used by json_serializer for sending to the API. Don't return CouchDB's properties
+        return {k: v for k, v in dict(self).items() if k not in COUCH_KEYS}
+
+
+class DataValue(DocumentSchema, AsDictMixin):
+    # Use DocumentSchema because it conveniently handles optional properties
```

Comment:
I was just using it to explicitly name the properties (i.e. the schema). We aren't going to be saving these to Couch, which is the only reason for choosing it over `Document`. The "optional properties" was in comparison to the named tuple that this replaced, where you'd have to specify all the properties of the tuple when you instantiate a new one. Happy to use `Document` instead.